swagger.yaml

```diff
@@ -129,6 +129,174 @@ paths:
         description: "`Chart` token"
         required: true
         type: string
+  "/{account}/datasets/{dataset}/fields":
+    x-summary: Dataset Field Descriptions
+    get:
+      operationId: getDatasetFields
+      summary: Get fields for a Dataset
+      description: Returns a list of the latest fields for a `Dataset`
+      parameters:
+      - name: account
+        in: path
+        description: "`Account` (`Workspace` or `User`) username"
+        required: true
+        type: string
+      - name: dataset
+        in: path
+        description: "`Dataset` token"
+        required: true
+        type: string
+      responses:
+        '404':
+          description: "`Dataset` not found"
+          schema:
+            "$ref": "#/definitions/NotFound"
+        '401':
+          description: Unauthorized
+          schema:
+            "$ref": "#/definitions/Unauthorized"
+        '200':
+          description: "`DatasetField` collection response"
+          schema:
+            "$ref": "#/definitions/DatasetFields"
+  "/{account}/datasets/{dataset}/field_descriptions":
+    x-summary: Dataset Field Descriptions
+    get:
+      operationId: getDatasetFieldDescriptions
+      summary: Get field descriptions for a Dataset
+      description: Returns a list of field descriptions for a `Dataset`
+      parameters:
+      - name: account
+        in: path
+        description: "`Account` (`Workspace` or `User`) username"
+        required: true
+        type: string
+      - name: dataset
+        in: path
+        description: "`Dataset` token"
+        required: true
+        type: string
+      responses:
+        '404':
+          description: "`Dataset` not found"
+          schema:
+            "$ref": "#/definitions/NotFound"
+        '401':
+          description: Unauthorized
+          schema:
+            "$ref": "#/definitions/Unauthorized"
+        '200':
+          description: "`DatasetFieldDescriptions` collection response"
+          schema:
+            "$ref": "#/definitions/DatasetFieldDescriptions"
+    post:
+      operationId: createDatasetFieldDescriptions
+      summary: Create a field description for a Dataset
+      description: Create a field description for a `Dataset` with the provided parameters
+      parameters:
+      - name: parameters
+        in: body
+        description: Dataset field description parameters
+        required: true
+        schema:
+          "$ref": "#/definitions/CreateDatasetFieldDescriptionBody"
+      - name: account
+        in: path
+        description: "`Account` (`Workspace` or `User`) username"
+        required: true
+        type: string
+      - name: dataset
+        in: path
+        description: "`Dataset` token"
+        required: true
+        type: string
+      responses:
```

Comment:
Should we have Dataset field not found here as well?

Reply:
Oh, good catch. I'll update the Webapp PR, this one, and the developer.mode.com docs. :)

Reply:
It looks like it will be updated to include `BadRequest` as that is what we return when the field isn't found. I'll also add that to Update and Destroy as that's a possible response.